dbt/models/ephemeral/schema.yml

```diff
@@ -0,0 +1,3 @@
+models:
+  - name: ephemeral_vacant_land
```

Comment:
Note the slightly different schema: `ephemeral_vacant_land` instead of `ephemeral.vacant_land`, as CTEs can't have periods in them. I think this is perfectly fine, but we can always refactor (to keep the period) later if we really want.